dbt/models/location/location_census.sql

```diff
@@ -0,0 +1,3 @@
+{{ config(materialized='ephemeral', alias='census') }}
```

Comment:
And here's the query definition for a model with a non-unique table name that's using an alias to resolve the collision.